ACM_General/events/managers.py

```diff
@@ -30,4 +46,10 @@ def _create_event(self, **kwargs):
         return model;
 
     def create_event(self, **kwargs):
+        """
+        Calls the '_create_event' function.
```

Comment:
_create_event public wrapper which a user can overwrite to add extra functionality to the private function.